src/runner/browser-job.js

```diff
@@ -99,7 +102,7 @@ export default class BrowserJob extends EventEmitter {
 
     // API
     get hasQueuedTestRuns () {
-        return !!this.testRunQueue.length;
+        return !!this.testRunQueue.length || this.opts.quarantineMode && !this.done;
```

Comment:
I don't so like this fix because it adds some mishmash. With this fix the `getQueuedTestRuns` don't tell us "is there a test run in the queue", but it's "is there a test run in the queue or there are a running test in quarantine mode".

Maybe we should fix just `browserConnection`?
Now it's 
```js
_popNextTestRunUrl () {
        if (this.jobQueue.length) {
            var job = this.jobQueue[0];
            var url = job.popNextTestRunUrl();

            if (!job.hasQueuedTestRuns)
                this.jobQueue.shift();

            return url;
        }

        return null;
    }
```
This gets the next test run url from a `job` and remove the `job` from the `jobQueue`. I propose change the sequence:
```js
_popNextTestRunUrl () {
        if (this.jobQueue.length) {
            if (!job.hasQueuedTestRuns)
                this.jobQueue.shift();

           if (this.jobQueue.length) {
                var job = this.jobQueue[0];
                var url = job.popNextTestRunUrl();

                return url;
          }
        }

        return null;
    }
```

/cc @inikulin @AndreyBelym